Yosemite/Yosemite/Stores/TaxClassesStore.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+import Networking
+import Storage
+
+
+// MARK: - TaxClassesStore
+//
+public class TaxClassesStore: Store {
+
+    private lazy var sharedDerivedStorage: StorageType = {
+        return storageManager.newDerivedStorage()
+    }()
+
+    /// Registers for supported Actions.
+    ///
+    override public func registerSupportedActions(in dispatcher: Dispatcher) {
+        dispatcher.register(processor: self, for: ProductAction.self)
```

Comment:
maybe not `ProductAction`? (I've had this many times before, the app would crash 😆 )